wpilibNewCommands/src/main/native/include/frc2/command/Commands.h

```diff
@@ -213,10 +213,9 @@ CommandPtr Sequence(std::vector<CommandPtr>&& commands);
 /**
  * Runs a group of commands in series, one after the other.
  */
-template <std::convertible_to<CommandPtr>... CommandPtrs>
 [[nodiscard]]
-CommandPtr Sequence(CommandPtrs&&... commands) {
-  return Sequence(impl::MakeVector(std::forward<CommandPtrs>(commands)...));
+CommandPtr Sequence(OwnedCommand&&... commands) {
```

Comment:
iirc, concepts require `OwnedCommand auto&&...` in this context. https://en.cppreference.com/w/cpp/language/constraints